libbeat/publisher/processing/default.go

```diff
@@ -107,8 +111,19 @@ func MakeDefaultSupport(
 		if err := beatCfg.Unpack(&cfg); err != nil {
 			return nil, err
 		}
+		// don't try to "merge" the two lists somehow, if the supportFactory caller requests its own processors, use those
+		// also makes it easier to disable global processors if needed, since they're otherwise hardcoded
+		var rawProcessors processors.PluginConfig
+		// don't check the array directly, use HasField, that way processors can easily be bypassed with -E processors=[]
+		if fleetmode.Enabled() && !beatCfg.HasField("processors") {
+			log.Debugf("In fleet with processors specified, defaulting to global processors")
+			rawProcessors = fleetDefaultProcessors
+
```

Comment:
If I got it correctly, this case is when fleet mode is enabled and no processors are specified, so we need to fix the log message:
```suggestion
		if fleetmode.Enabled() && !beatCfg.HasField("processors") {
			log.Debugf("In fleet mode with no processors specified, defaulting to global processors")
			rawProcessors = fleetDefaultProcessors
```